x-pack/test/functional/services/index.ts

```diff
@@ -46,7 +46,20 @@ import { UserMenuProvider } from './user_menu';
 import { UptimeProvider } from './uptime';
 import { InfraSourceConfigurationFormProvider } from './infra_source_configuration_form';
 import { InfraLogStreamProvider } from './infra_log_stream';
-import { MachineLearningProvider } from './machine_learning';
+import {
+  MachineLearningAnomalyExplorerProvider,
+  MachineLearningAPIProvider,
+  MachineLearningDataFramesProvider,
+  MachineLearningDataVisualizerProvider,
+  MachineLearningJobManagementProvider,
+  MachineLearningJobSourceSelectionProvider,
+  MachineLearningJobTypeSelectionProvider,
+  MachineLearningJobWizardCommonProvider,
+  MachineLearningNavigationProvider,
+  MachineLearningSettingsProvider,
+  MachineLearningSingleMetricViewerProvider,
+  // @ts-ignore not ts yet
```

Comment:
This module actually is TS, right?

Reply:
Right, fixed together with the ML services restructuring.